docs/guides/modules/orchestrate/pages/set-up-multiple-configuration-files-for-a-project.adoc

```diff
@@ -42,51 +42,47 @@ NOTE: **Splitting up an existing configuration?** If you are following these ste
 . Complete the required form fields:
 ** Give your pipeline a descriptive name.
-** Ensure **GitHub App** is selected under **Integration**.
-** If you see a btn:[Connect] button, select it and follow instructions to install the GitHub App for your org. You will need to be an org admin to install the GitHub App.
-** Select the same repository as the project you are building under **Config Source**. Specify where your config file is located under **Config File Path**.
-** Under **Build Assets** select your code source under **Checkout Source** -- this is the code that will be checked out when using the xref:reference:ROOT:configuration-reference.adoc#checkout[`checkout` step] in your config file.
+** Select the same repository as the project you are building under **Config Source**.
+** Specify where your config file is located under **Config File Path**.
+** Select your code source under **Checkout Source** -- this is the code that will be checked out when using the xref:reference:ROOT:configuration-reference.adoc#checkout[`checkout` step] in your config file.
 . Select btn:[Save]
 +
-image::guides:ROOT:add-pipeline.png[Add a pipeline page in CircleCI app]
+image::guides:ROOT:orchestrate-and-trigger/project-setup-pipelines.png[Add a pipeline]
 
 [#create-a-new-trigger]
 == 3. Create a new trigger
 
-Next, create a trigger for each pipeline you have created for your project. Alternatively, you can trigger a pipeline of your choosing manually xref:triggers-overview.adoc#run-a-pipeline-from-the-circleci-web-app[via the web app].
+Next, create a trigger for each pipeline you have created for your project. Alternatively, you can trigger a pipeline manually xref:triggers-overview.adoc#run-a-pipeline-from-the-circleci-web-app[via the web app].
 
-. Following from the previous step, select **Triggers** in the sidebar
-+
-. Select btn:[Add Trigger].
-. Select **GitHub App** in the dropdown menu.
-. Select btn:[Next].
-. Complete the required form fields:
-** Give your trigger a descriptive name.
-** Select your project repository from the dropdown.
-** Choose your new pipeline from the "Choose config to run" menu.
-** Choose a trigger event options from the "Run on" menu.
+. Following from the previous step, select btn:[GitHub trigger +] at the bottom of the pipeline box.
+. Choose when top run the pipeline by selecting an option from the *Event* menu.
 +
 ****
-Using multiple pipelines and triggers, you can build  different pipelines for different events, For example, you could have:
+Using multiple pipelines and triggers, you can build different pipelines for different events, For example, you could have:
```

Comment:
Should this link to https://circleci.com/docs/guides/orchestrate/set-up-multiple-configuration-files-for-a-project/ ?